src/prefect/types/_datetime.py

```diff
@@ -37,6 +39,14 @@ def local_timezone() -> Timezone | FixedTimezone:
     return pendulum.tz.local_timezone()
 
 
+def get_timezones() -> tuple[str, ...]:
+    return pendulum.tz.timezones()
+
+
+def pendulum_instance(v: DateTime) -> DateTime:
```

Comment:
Could we name this something like `create_datetime_instance`? Also, why do we need to call `.instance` twice?

Reply:
yep happy to rename but the double .instance is because `not isinstance(pendulum.DateTime, DateTime)`